src/app/caupona-content/calendar/calendar-settings/calendar-settings.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="flex flex-column">
+  <label>Google calendar integration</label>
+  <mat-radio-group
+    [(ngModel)]="googleintegration"
+    (change)="setGoogleIntegration()"
+  >
+    <mat-radio-button [value]="true">Turn on</mat-radio-button>
```

Comment:
to jest wgl potrzebne? jeśli tak to zrób jakąś zmienną/pole w klasie, żeby nie było na sztywno true/false

Reply:
Nie mam makiety, żeby przebudować ten widok także w przyszłości to najprawdopodobniej zostanie wyrzucone.